howso/hyperparameters.amlg

```diff
@@ -816,15 +830,22 @@
 	#set_auto_analyze_params
 	(declare
 		(assoc
+			;{type "boolean"}
 			auto_analyze_enabled (false)
+			;{type "number"}
 			analyze_threshold 100
+			;{type "number"}
 			analyze_growth_factor 7.389056
+			;{type "number"}
 			auto_analyze_limit_size 200000
+		)
+		(call !ValidateParameters)
 
+		(declare (assoc
```

Comment:
alright, so this method sucks in that actually it allows all the parameters to analyze to be specified as optional. notice how they are all being set on lines ~892-910 below, and these were actually meant to be as part of the input except they explicitly needed default values instead of just being null.

I think you'll have to copy them all from analyze into this block as well.

Reply:
done